CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ See [SETUP.md](./SETUP.md) for instructions on how to get everything running.
 
 ## Picking Up Tickets
 
-My todo list for this application can be found [here](https://github.com/users/scottbenton/projects/5).
+My todo list for this application can be found [here](https://github.com/users/scottbenton/projects/7/views/3).
```

Comment:
Thanks for fixing this (and adding to the setup file)! Copied it over from the other project and forgot to update it.